x-pack/plugins/security_solution/public/resolver/data_access_layer/factory.ts

```diff
@@ -64,7 +64,7 @@ export function dataAccessLayerFactory(
       const response: ResolverPaginatedEvents = await context.services.http.post(
         '/api/endpoint/resolver/events',
         {
-          query: {},
+          query: { limit: 1 },
```

Comment:
❔ Provided that we know it's possible (IIRC) to have more than 1 event with the same event.id under certain circumstances, is that stable? E.g. What if there were 500 events with that ID - could you get different ones by calling this?

Reply:
it might make sense to pull back all events with the same event id. The data access layer was already only reading the first event, so this shouldn't hurt.

If we pulled back all events with the same ID then the event details view would need to handle that somehow.

Reply:
Yeah probably better to cap that off here, I guess. I just wanted to check.